docs/en/ingest-management/elastic-agent/ingest-pipeline-in-Kubernetes.asciidoc

```diff
@@ -0,0 +1,83 @@
+[[ingest-pipeline-in-Kubernetes]]
+= Using Custom Ingest Pipeline with {k8s} Integration
+
+This tutorial explains how to add a custom ingest pipeline to {k8s} Integration in order to add specific metadata fields for deployments and cronjobs of pods.
+
+Custom pipelines can be used to add custom data processing, like adding fields, obfuscate sensitive information, and more. Find more information in [data-streams-pipeline-tutorial](https://www.elastic.co/guide/en/fleet/master/data-streams-pipeline-tutorial.html) on how to use custom ingest pipelines.
+
+== Metadata Enrichment for Kubernetes
+
+The https://docs.elastic.co/en/integrations/kubernetes[{k8s} Integration] is used to collect logs and metrics from Kubernetes clusters with {agent}. During the collection, integration enhances the collected information with extra useful information that can help users to correlate information and different Kubernetes assets with each other. All this additional information that are added on top of collected information like labels, annotations, ancestor names of Kubernetes assets etc. are called metadata.
+
+The https://www.elastic.co/guide/en/fleet/current/kubernetes-provider.html[{k8s} Provider] offers the `add_resource_metadata` option to configure the metadata enrichment options.
+
+For {agent} versions >[8.10.4], the default configuration for metadata enrichment is `add_resource_metadata.deployment=false` and `add_resource_metadata.cronjob=false`. This means that pods that created from replicasets that belong to specific deployments would not be enriched with `kubernetes.deployment.name`. Additionally, pods that are created from jobs that belong to specific cronjobs, would not be enriched with `kubernetes.cronjob.name`.
+
+**Kubernetes Integration Policy > Collect Kubernetes metrics from Kube-state-metrics > Kubernetes Pod Metrics**
+
+--
+[role="screenshot"]
+image::images/add_resource_metadata.png[Configure add_resource_metadata]
```

Comment:
It looks like this image is missing. Adding it in might fix the failing docs CI check.

Reply:
My mistake added this